src/entropic/greenkhorn.jl

```diff
@@ -0,0 +1,114 @@
+# Greenkhorn is a greedy version of the Sinkhorn algorithm
+# This method is from https://arxiv.org/pdf/1705.09634.pdf
+# Code is based on implementation from package POT
+
+
+"""
+    Greenkhorn()
+
+Greenkhorn is a greedy version of the Sinkhorn algorithm.
+"""
+struct Greenkhorn <: Sinkhorn end
+
+struct GreenkhornCache{U,V,KT}
+    u::U
+    v::V
+    K::KT
+    Kv::U #placeholder
```

Comment:
Why "placeholder"?

Reply:
This is a partial solution. Without this `Kv`, I got an error. It seems that  the `Sinkhorn` structs further on require it. I could not find out how to get rid of it without changing the code for Sinkhorn.

Reply:
I mean you use it explicitly below for checking convergence?

Reply:
Yeah, it's been used to check the convergence. But I think the convergence check might be done another way more efficiently. But I was having trouble getting it to work with the existing "api" for Sinkhorn, so I updated `Kv` and used the convergence verification already in place.

You are right that, as is, `Kv` is not actually just a placeholder, since it's been used in the convergence.